go/integration_test/processor/create_test.go

```diff
@@ -633,7 +633,92 @@ func TestProcessor_CodeStartingWith0xEFCanNotBeCreated(t *testing.T) {
 					if result.GasUsed != sufficientGas {
 						t.Errorf("execution failed but gas was not fully used, used %d", result.GasUsed)
 					}
+				}
+			})
+		}
+	}
+}
+
+func TestProcessor_InitCodeSizeIsLimited(t *testing.T) {
+	maxInitCodeSize := int64(24576)
```

Comment:
According to [EIP-3860](https://eips.ethereum.org/EIPS/eip-3860) the maximum init code size is 49152 = 2x the maximum code size. Is this constant off?